azure-pipelines.release.yml

```diff
@@ -290,6 +306,8 @@ stages:
                 value: $(service_web_url_staging)
               active:
                 value: $(service_web_url)
+          - name: VhServices:EnableMobileSupport
```

Comment:
this key is not added in azurepipelines.yml and .nightly.yml

Reply:
can we set that to false by default in app config?